Dockerfile

```diff
@@ -3,49 +3,49 @@
 FROM debian:jessie
 MAINTAINER Open Knowledge
 
-ENV CKAN_HOME /usr/lib/ckan/default
+ENV CKAN_HOME /usr/lib/ckan
+ENV CKAN_VENV $CKAN_HOME/default
 ENV CKAN_CONFIG /etc/ckan/default
 ENV CKAN_STORAGE_PATH /var/lib/ckan
 ENV CKAN_SITE_URL http://localhost:5000
 
+ENV DEBIAN_FRONTEND noninteractive
+
 # Install required packages
-RUN apt-get -q -y update && apt-get -q -y upgrade && DEBIAN_FRONTEND=noninteractive apt-get -q -y install \
-		python-dev \
-        python-pip \
-        python-virtualenv \
-        libpq-dev \
-        git-core \
-	&& apt-get -q clean
+RUN apt-get -q -y update && apt-get -q -y upgrade && apt-get -q -y install \
+    python-dev \
+    python-pip \
+    python-virtualenv \
+    libpq-dev \
+    git-core \
+    postgresql-client \
+    && apt-get -q clean && rm -rf /var/lib/apt/lists/*
+
+# add ckan user which runs all the ckan related stuff
+RUN useradd -r -u 900 -m -c "ckan account" -d $CKAN_HOME -s /bin/false ckan
```

Comment:
I've checked your patch and in your case the issue exists because you have another paths:

```
ENV CKAN_HOME /usr/lib/ckan/default
ENV CKAN_VENV $CKAN_HOME/default
```

that's why it can not add user since /usr/lib/ckan/ does not exist